tools/operator/launcher/k8s/network.go

```diff
@@ -2,241 +2,522 @@ package k8s
+	envs = append(envs, env)
+	env = v12.EnvVar{Name: "FABRIC_CA_SERVER_TLS_CERTFILE", Value: fmt.Sprintf("/etc/hyperledger/fabric/artifacts/tlsca.%s-cert.pem", orgName)}
+	envs = append(envs, env)
+	env = v12.EnvVar{Name: "FABRIC_CA_SERVER_CA_NAME", Value: fmt.Sprintf("ca%d-%s", id, orgName)}
+	envs = append(envs, env)
+	container.Env = envs
+	volumeMount := v12.VolumeMount{MountPath: "/etc/hyperledger/fabric/artifacts/", Name: "cacerts"}
+	container.VolumeMounts = append(container.VolumeMounts, volumeMount)
+	containers = append(containers, container)
+	l.Containers = containers
+	volume := v12.Volume{Name: "cacerts", VolumeSource: v12.VolumeSource{ConfigMap: &v12.ConfigMapVolumeSource{LocalObjectReference: v12.LocalObjectReference{Name: fmt.Sprintf("%s-ca", orgName)}}}}
+	l.Volumes = append(l.Volumes, volume)
+	return l
+}
+
+func (k8s K8s) volumeMountLists(componentType, dataPersistence string, enableNodeOUs bool) []v12.VolumeMount {
+
+	volumeMounts := make([]v12.VolumeMount, 0)
+	volumeMount := v12.VolumeMount{MountPath: "/etc/hyperledger/fabric/artifacts/msp/admincerts/", Name: "admincerts"}
+	volumeMounts = append(volumeMounts, volumeMount)
+	volumeMount = v12.VolumeMount{MountPath: "/etc/hyperledger/fabric/artifacts/msp/cacerts/", Name: "cacerts"}
+	volumeMounts = append(volumeMounts, volumeMount)
+	volumeMount = v12.VolumeMount{MountPath: "/etc/hyperledger/fabric/artifacts/msp/signcerts/", Name: "signcerts"}
+	volumeMounts = append(volumeMounts, volumeMount)
+	volumeMount = v12.VolumeMount{MountPath: "/etc/hyperledger/fabric/artifacts/msp/keystore/", Name: "keystore"}
+	volumeMounts = append(volumeMounts, volumeMount)
+	volumeMount = v12.VolumeMount{MountPath: "/etc/hyperledger/fabric/artifacts/msp/tlscacerts/", Name: "tlscacerts"}
+	volumeMounts = append(volumeMounts, volumeMount)
+	volumeMount = v12.VolumeMount{MountPath: "/etc/hyperledger/fabric/artifacts/tls/", Name: "tls"}
+	volumeMounts = append(volumeMounts, volumeMount)
+	if enableNodeOUs {
+		volumeMount = v12.VolumeMount{MountPath: "/etc/hyperledger/fabric/artifacts/msp/", Name: "config"}
+		volumeMounts = append(volumeMounts, volumeMount)
 	}
-	args = append(args, componentName)
-	for i := 0; i < len(k.Arguments); i++ {
-		switch k.Action {
-		case "create":
-			args = append(args, fmt.Sprintf("--from-file=%s", k.Arguments[i]))
-		default:
-			args = append(args, k.Arguments[i])
-		}
+	volumeMount = v12.VolumeMount{MountPath: "/etc/hyperledger/fabric/", Name: fmt.Sprintf("%s-config", componentType)}
+	volumeMounts = append(volumeMounts, volumeMount)
+	if dataPersistence == "true" || dataPersistence == "local" {
+		volumeMount = v12.VolumeMount{MountPath: "/shared/data/", Name: "data-storage"}
+		volumeMounts = append(volumeMounts, volumeMount)
+	}
+	if componentType == "orderer" {
+		volumeMount = v12.VolumeMount{MountPath: "/etc/hyperledger/fabric/genesisblock", Name: "genesisblock"}
+		volumeMounts = append(volumeMounts, volumeMount)
+	}
+	return volumeMounts
+}
```

Comment:
We can improve the readability of this function:

```
func (k8s K8s) volumeMountLists(componentType, dataPersistence string, enableNodeOUs bool) []corev1.VolumeMount {
	volumeMounts := []corev1.VolumeMount{
		{MountPath: "/etc/hyperledger/fabric/artifacts/msp/admincerts/", Name: "admincerts"},
		{MountPath: "/etc/hyperledger/fabric/artifacts/msp/cacerts/", Name: "cacerts"},
		{MountPath: "/etc/hyperledger/fabric/artifacts/msp/signcerts/", Name: "signcerts"},
		{MountPath: "/etc/hyperledger/fabric/artifacts/msp/keystore/", Name: "keystore"},
		{MountPath: "/etc/hyperledger/fabric/artifacts/msp/tlscacerts/", Name: "tlscacerts"},
		{MountPath: "/etc/hyperledger/fabric/artifacts/tls/", Name: "tls"},
		{MountPath: "/etc/hyperledger/fabric/", Name: fmt.Sprintf("%s-config", componentType)},
	}
	if enableNodeOUs {
		volumeMount := corev1.VolumeMount{MountPath: "/etc/hyperledger/fabric/artifacts/msp/", Name: "config"}
		volumeMounts = append(volumeMounts, volumeMount)
	}
	if dataPersistence == "true" || dataPersistence == "local" {
		volumeMount := corev1.VolumeMount{MountPath: "/shared/data/", Name: "data-storage"}
		volumeMounts = append(volumeMounts, volumeMount)
	}
	if componentType == "orderer" {
		volumeMount := corev1.VolumeMount{MountPath: "/etc/hyperledger/fabric/genesisblock", Name: "genesisblock"}
		volumeMounts = append(volumeMounts, volumeMount)
	}
	return volumeMounts
}
```

Reply:
Yes I agree.. I also felt to change to something with more readability. Thank you. Fixed.